packages/slice-machine/src/modules/selectedSlice/screenshot.ts

```diff
@@ -0,0 +1,61 @@
+import { fetchApi } from "@lib/builders/common/fetch";
+import { ScreenshotUI } from "@lib/models/common/ComponentUI";
+import { ScreenshotResponse, Screenshots } from "@models/common/Screenshots";
+
+export function generateScreenShot(
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  _variationId: string,
+  libraryName: string,
+  sliceName: string,
+  setData: (data: any) => void,
+  callback: (screenshots: Screenshots) => void
+) {
+  // eslint-disable-next-line @typescript-eslint/no-floating-promises
+  fetchApi({
```

Comment:
return, await or void if you don't want to wait for the result :)

Reply:
yes, changed here and for the other fetchApi calls